### PR TITLE
Make tab header fix when scrolling in param dialogue

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

**Review by ayolab — APPROVED** [review]

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 1]
```diff
@@ -299,8 +299,8 @@ export const LoadFlowParameters = ({
     }, [resetParameters, resetProvider]);
 
     return (
-        <Grid container className={classes.grid}>
-            <Grid container key="provider">
+        <>
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/load-flow-parameters.js:302]: Why changing <Grid container className={classes.grid}> to <> ?

File: src/components/dialogs/parameters/map-parameters.js [hunk 2]
```diff
@@ -72,103 +72,105 @@ export const MapParameters = ({ hideParameters }) => {
...
-                </Typography>
-            </Grid>
-            <Grid item container xs={4} className={classes.controlItem}>
-                <Select
-                    size="small"
-                    labelId="line-flow-mode-select-label"
-                    value={lineFlowModeLocal}
-                    onChange={(event) => {
-                        handleChangeLineFlowMode(event.target.value);
+        <>
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/map-parameters.js:75]: Same. Why this change ?

File: src/components/dialogs/parameters/network-parameters.js [hunk 3]
```diff
@@ -52,8 +52,8 @@ export const NetworkParameters = ({ hideParameters }) => {
     };
 
     return (
-        <Grid container spacing={1} className={classes.grid}>
-            <Grid item container spacing={1}>
+        <>
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/network-parameters.js:55]: same

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 4]
```diff
@@ -299,8 +299,8 @@ export const LoadFlowParameters = ({
...
+            <Grid container className={classes.scrollableGrid} key="provider">
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/load-flow-parameters.js:303]: This is not a problem of you PR. But you can change key="provider" to key="basicParameters" nope ?

File: src/components/dialogs/parameters/parameters.js [hunk 5]
```diff
@@ -127,14 +127,15 @@ export const useStyles = makeStyles((theme) => ({
     },
     grid: {
         paddingTop: theme.spacing(2),
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/parameters.js:129]: You can remove this one

File: src/components/dialogs/parameters/parameters.js [hunk 6]
```diff
@@ -474,13 +486,14 @@ const Parameters = ({ user, isParametersOpen, hideParameters }) => {
...
             </DialogTitle>
-            <DialogContent>
+            <DialogContent style={{ overflowY: 'hidden' }}>
                 <Container maxWidth="md">
                     <Tabs
                         value={tabValue}
                         variant="scrollable"
                         onChange={(event, newValue) => setTabValue(newValue)}
                         aria-label="parameters"
+                        style={{ position: 'sticky', top: 0 }}
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/parameters.js:491]: Do you need this style ?

Comment (YenguiSeddik) [reply src/components/dialogs/parameters/parameters.js]: i removed it

**Review by etiennehomer — APPROVED** [review]
